src/test/coll.rs

```diff
@@ -25,6 +25,65 @@ use crate::{
     RUNTIME,
 };
 
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+#[function_name::named]
+async fn insert_err_info() {
+    let _guard = LOCK.run_exclusively().await;
+
+    let client = TestClient::new().await;
+    let coll = client
+        .init_db_and_coll(function_name!(), function_name!())
+        .await;
+    if !client.is_replica_set() {
+        return;
+    }
+    client
+        .database("admin")
+        .run_command(
+            doc! {
+                "configureFailPoint": "failCommand",
+                "data": {
+                "failCommands": ["insert"],
+                "writeConcernError": {
+                    "code": 100,
+                    "codeName": "UnsatisfiableWriteConcern",
+                    "errmsg": "Not enough data-bearing nodes",
+                    "errInfo": {
+                    "writeConcern": {
+                        "w": 2,
+                        "wtimeout": 0,
+                        "provenance": "clientSupplied"
+                    }
+                    }
+                }
+                },
+                "mode": { "times": 1 }
+            },
+            None,
+        )
+        .await;
```

Comment:
The clippy warning in CI is referring to this line; we're running a command and returning a Result, but we're not using it. Since this command need to be successfully for the test to work, you should `unwrap` here.